File: 2019/j1.py
# code by savir singh

a3 = int(input())
a2 = int(input())
a1 = int(input())
b3 = int(input())
b2 = int(input())
b1 = int(input())

ascore = 3*a3+2*a2+a1
bscore = 3*b3+2*b2+b1

if ascore > bscore:
    print("A")
elif bscore > ascore:
    print("B")
else:
    print("T")

File: 2019/j1.cpp
// code by savir singh

#include <bits/stdc++.h>

using namespace std;

int main() {
    int a, b, c;
    cin >> a >> b >> c;
    int ascore=a*3+b*2+c;
    int d, e, f;
    cin >> d >> e >> f;
    int bscore=d*3+e*2+f;
    if (ascore>bscore) {
        cout << "A";
    }
    else if (bscore>ascore) {
        cout << "B";
    }
    else {
        cout << "T";
    }
}
